src/components/pages/Categories/CategoryMain.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <section class="category-list">
+    <div class="category">
+      <app-category-post
+        class="category__post"
+        :category="name"
+      />
+      <app-category-list
+        class="category__list"
+        :theads="theads"
+        :categories="categoryList"
+      />
+    </div>
+  </section>
+</template>
+
+<script>
+import { CategoryList, CategoryPost } from '@Components/molecules';
+// import Mixins from '@Helpers/mixins';
+
+export default {
+  components: {
+    appCategoryList: CategoryList,
+    appCategoryPost: CategoryPost,
+  },
+  data() {
+    return {
+      theads: ['カテゴリー名'],
+      name: '',
+    };
+  },
+  computed: {
+    categoryList() {
+      return this.$store.state.categories.categories;
+    },
+  },
+  created() {
+    this.$store.dispatch('categories/getAllCategories');
+  },
+  methods: {
+    createCategory() {
+      if (this.loading) return;
+      this.$store.dispatch('users/createCategory', {
+        name: this.name,
+      }).then(() => {
+      });
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
```

Comment:
[nits]

空行追加した方が見やすいです！

```suggestion
</script>

<style lang="scss" scoped>
```

Reply:
7e18b89dd260353aee4a132223ab82fa44624139

41行目の</script>と<style>間に空行を追加しました！